ocaml/forkexecd/helper/dune

```diff
@@ -0,0 +1,27 @@
+(rule
```

Comment:
Following dune's [docs about non-ocaml artifacts](https://dune.readthedocs.io/en/stable/foreign-code.html#foreign-sandboxing), I think this file can be replaced with a smaller file, `ocaml/forkexecd/dune`:
```lisp
(data_only_dirs helper)

(rule
  (deps (source_tree helper))
  (targets vfork_helper)
  (package forkexec)
  (action
    (no-infer
      (progn
        (chdir helper (run make))
        (copy helper/vfork_helper vfork_helper)
      )
    )
  )
)

(install
  (package forkexec)
  (section libexec_root)
  (files (vfork_helper as xapi/vfork_helper))
)
```

Reply:
I'll try.

Reply:
With some external adaptation it works. But why is this version better? The main shrunk came from removing the explicit dependency files from the dune file which to be honest looks better to me.

Reply:
dune works per-directory. This way, the directory is managed by an external build tool exclusively (make)

Reply:
done